packages/table/src/cell/cell.tsx

```diff
@@ -102,7 +102,15 @@ export class Cell extends React.Component<ICellProps, {}> {
             },
         );
 
-        const content = <div className={textClasses}>{this.props.children}</div>;
+        const modifiedChildren = React.Children.map(this.props.children, (child) => {
+            if (React.isValidElement(child)) {
```

Comment:
hmm i didn't know this method existed. is it necessary? does it actually avoid errors? my suspicion is no, and you don't need it.

Reply:
It's a typeguard. Without it, child could be react element, or could be text. Hence the return, otherwise.